lib/quadquizaminos_web/live/tetris_live.ex

```diff
@@ -408,13 +414,41 @@ defmodule QuadquizaminosWeb.TetrisLive do
     {:noreply, socket}
   end
 
-  def handle_event("add_block", %{"x" => x, "y" => y}, socket) do
+  def handle_event("move_block", %{"x" => x, "y" => y, "color" => color}, %{assigns: %{moving_block: true}} = socket) do
+    x = String.to_integer(x)
```

Comment:
Future refactor we can have this as a separate function, we have it repeated more than 3 times